src/components/FormSectionHeader/FormSectionHeader.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Grid, makeStyles } from '@material-ui/core';
+
+const borderColor = '#404040';
+const useStyles = makeStyles({
+  titleBox: {
+    fontFamily: 'Avenir Next',
+    backgroundColor: borderColor,
+    color: 'white',
+    border: `3px solid ${borderColor}`,
+    letterSpacing: 1,
+    display: 'inline-block'
+  },
+  line: {
+    display: 'inline-block',
+    border: `1px solid ${borderColor}`,
+    width: '100%'
+  }
+});
+
+function FormSectionHeader({ title }) {
+  const classes = useStyles();
+
+  return (
+    <Grid container direction="row">
+      <Grid item>
+        <div className={classes.titleBox}>{title}</div>
+      </Grid>
+      <Grid item xs={9}>
+        <div className={classes.line}></div>
```

Comment:
This line should take up 100% of the `FormSectionHeader` container. Currently all the lines are different lengths. Consider using flex box instead of the Grid components here.

Reply:
I switched to flexbox. Never used it before, so let me know if I did it relatively correctly haha